### PR TITLE
Update content of Design page, fix ##655

### PR DESCRIPTION
**Pull request by jancborchardt**
Please review @skjnldsv @MorrisJobke @juliushaertl @pixelipo @danxuliu @Espina2 @jospoortvliet – and say if we should change anything else. :)

The next step would be to change the layout to the proper page with nice big image in header. :) So that we can have this be the landing page for prospective design contributors. With 3 columns for example: Get involved, Team, Guidelines.

### CODE REVIEW

**Comment by jospoortvliet** [comment]
Looking good, agreed on the new design thingy. If you give me a header image I can do the basic work on the code side, if you like.

**Comment by jospoortvliet** [comment]
also, https://github.com/nextcloud/nextcloud.com/pull/670 - how about you check and merge that one ;-)

File: page-design.php [hunk 1]
```diff
@@ -1,70 +1,72 @@
...
-	<li><a href="https://github.com/enoch85">Daniel Hansson</a></li>
-	<li><a href="https://github.com/stefan-niedermann">Stefan Niedermann</a></li>
-	<li><a href="https://github.com/Espina2">Paulo moura</a></li>
-	<li><?php echo $l->t('and many others, see <a href="https://github.com/orgs/nextcloud/teams/designers">here</a>.');?></li>
+	<li><a href="https://github.com/jancborchardt" target="_blank">Jan-Christoph Borchardt</a> (Design lead)</li>
+	<li><a href="https://github.com/skjnldsv" target="_blank">John Molakvoæ</a> (frontend, documentation &amp; Contacts app)</li>
+	<li><a href="https://github.com/MorrisJobke" target="_blank">Morris Jobke</a> (frontend)</li>
+	<li><a href="https://github.com/juliushaertl" target="_blank">Julius Härtl</a> (frontend &amp; Deck app)</li>
+	<li><a href="https://github.com/pixelipo" target="_blank">Marin Treselj</a> (frontend &amp; Deck app)</li>
+	<li><a href="https://github.com/https://github.com/danxuliu" target="_blank">Daniel Calviño Sánchez</a> (frontend &amp; Talk app)</li>
```
[the remark this answers is not in the document]
Comment (jancborchardt) [reply page-design.php]: Thanks! Committed directly https://github.com/nextcloud/nextcloud.com/commit/87f153a609c771e58f3b88f9cab51930bec14667

**Comment by jancborchardt** [comment]
> Looking good, agreed on the new design thingy. If you give me a header image I can do the basic work on the code side, if you like.

That would be very cool! As image we could start with this one, which is a design sketching pic taken from a 2012 event of ours ;)
design sketching
[Source jpg](https://cloud.nextcloud.com/s/58Qm8ayzVuC946W)

**Comment by jospoortvliet** [comment]
Sounds like a plan, expect a PR tomorrow or the day after (it is late here, about to go sleep and tomorrow the wedding ceremonies start, lasting 3 days).

**Pull request merged**